.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+config.properties
```

Comment:
Make sure that ignoring `config.properties` is intentional. If this file is needed for configuration in different environments, consider using environment-specific configuration management instead of ignoring it globally.